src/test-cases-app/test-cases-app.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ReactTestKitchen from '@mapbox/react-test-kitchen';
+import componentIndex from './component-index';
```

Comment:
You'll want to `// eslint-disable-line` this import in case ESLint runs at a point when that file isn't generated.